modules/core/src/lib/deck.js

```diff
@@ -813,6 +823,14 @@ export default class Deck {
     });
   }
 
+  _onWebGLContextLost(event) {
+    if (this.props.onError) {
+      this.props.onError(event);
+    }
+  }
+
+  _onWebGLContextRestored(event) {}
```

Comment:
Is this necessary? Leave a TODO comment if you are leaving certain implementation out of this PR, otherwise remove.